### PR TITLE
Fix/node 17 hash compatibility

### PR DESCRIPTION
**Pull request by maartenst**
## What I did

1.Change the hashing for rollup-plugin-html from md4 to md5 to be compatible with Node 17 and up.
2.Add 'yarn build' to the Contributing file setup stage to avoid it being missed before trying to run tests

### CODE REVIEW

File: .github/workflows/verify-node.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
           - '12'
```
Comment (LarsDenBakker) [inline .github/workflows/verify-node.yml:15]: Let's drop running node 12 tests (it goes out of support after april) to avoid making the CI slower

**Comment by maartenst** [comment]
waiting for https://github.com/modernweb-dev/web/pull/1926 to be merged and then this one probably works as well

**Comment by daKmoR** [comment]
@maartenst need some help with a rebase?

**Comment by daKmoR** [comment]
thx @maartenst I move the code over to https://github.com/modernweb-dev/web/pull/1937 and adjusted 2 more tests to make it pass the CI 👍

took a little fiddling that's why I did it in a separate PR

**Pull request closed without merging**